Sorting/sorting.cpp

```diff
@@ -20,7 +20,12 @@ void Sort::GetInput(){
 }
 
 /**
- * Print.
+ * Depending upon the boolean input,prints the corresponding array separated by comma.When the input is true
+ * it prints the sorted array and when it is false it prints the unsorted array.
+ * 
+ * Inputs: boolean value True or False
+ * 
+ * Returns: null
```

Comment:
This is to be done in a separate PR. Revert.
Please proceed type by type, from 1 to 4, in 4 separate PRs.